### PR TITLE
Use ZVAL_COPY_VALUE when passing parameters

### PR DESCRIPTION
**Pull request by sjinks**

### CODE REVIEW

File: kernels/ZendEngine3/fcall.c [hunk 1]
```diff
@@ -283,8 +283,7 @@ int zephir_call_user_function(zval *object_pp, zend_class_entry *obj_ce, zephir_
 
 	zval p[fci.param_count];
 	for (i=0; i<fci.param_count; ++i) {
-		/** FIXME: ZVAL_COPY or ZVAL_COPY_VALUE? */
-		ZVAL_COPY(&p[i], params[i]);
+		ZVAL_COPY_VALUE(&p[i], params[i]);
```
Comment (sergeyklay) [inline kernels/ZendEngine3/fcall.c:286]: You have a good memory :smiley:

Comment (sjinks) [reply kernels/ZendEngine3/fcall.c:286]: Just experience :-) I had a similar issue in PHP-CPP

Comment (sjinks) [reply kernels/ZendEngine3/fcall.c:286]: "Parameter 3 to preg_match_all() expected to be a reference, value given" - this was the clue

**Comment by sergeyklay** [comment]
Is it ready to merging?

**Comment by sergeyklay** [comment]
Good catch! Thank you 👍

**Pull request merged**